lib/rules/tap-consistent-assertions.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const synonyms = require('./common/tap-synonyms.js')
```

Comment:
Is it possible to scope this to a directory? 

I would only want these rule to apply to tests

Reply:
I looked into this way back and didn't find any way to scope it to a directory at runtime, but I'll take another peek. worst case we could run this with a separate config, or even command `eslint --rule ...`

Reply:
yeah it's actually easy to do this via config [`overrides`](https://eslint.org/docs/user-guide/configuring#specifying-processor), and flipping on rules for certain paths